x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/alert_actions.tsx

```diff
@@ -114,13 +85,20 @@ export function AlertActions({
     closeActionsPopover();
   };
 
-  const handleUntrackAlert = useCallback(async () => {
-    await untrackAlerts({
-      indices: [ecsData?._index ?? ''],
-      alertUuids: [alertId],
-    });
-    onSuccess();
-  }, [untrackAlerts, alertId, ecsData, onSuccess]);
+  const DefaultAlertActions = useMemo(
+    () =>
+      triggersActionsUi?.getAlertsTableDefaultAlertActions({
+        key: 'defaultRowActions',
+        onActionExecuted: closeActionsPopover,
+        isAlertDetailsEnabled: false,
+        resolveRulePagePath: (alertRuleId) =>
+          alertRuleId
+            ? `/app/management/insightsAndAlerting/triggersActions/rule/${alertRuleId}`
+            : null,
```

Comment:
❓  You still pass an `alert` object to `getAlertsTableDefaultAlertActions`. What is the reason for having `resolveRulePagePath`? I reckon it should happen inside of this function, and the callback seems redundant.